ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -211,3 +168,43 @@ class StandaloneIndexServer(
   }
 
 }
+
+object StandaloneApiServer {
+  private val actorSystemName = "index"
+
+  private val asyncTolerance = 30.seconds
+
+  private val sharedEngine = Engine()
+
+  private final class ApiServerState(
+      ledgerId: LedgerId,
+      apiServer: ApiServer,
+      indexAndWriteService: AutoCloseable,
+  ) extends AutoCloseable {
+    def port: Int = apiServer.port
+
+    override def close(): Unit = {
+      apiServer.close() //fully tear down the old server.
```

Comment:
If I knew, I would have improved the code and deleted the comment, but I have no idea. This is just a move from above.

Reply:
I'm not sure of the meaning either and in any case it doesn't seem very informative. I _suppose_ someone with some context on how things were handled in some previous version of this can extrapolate some information from this but I don't think it's the case anymore, so feel free to remove it if you want.

Reply:
Agreed.

Reply:
This comment is misleading as it talks about the "old server" but nowhere mentions a new one. Can be deleted, because `close` just closes the api server. period.

Reply:
Deleted.